src/otserv.cpp

```diff
@@ -141,105 +108,111 @@ void mainLoader(int, char*[], ServiceManager* services)
 		c_test.close();
 	}
 
-	// read global config
+	// Load '/config.lua' into 'g_config'
 	std::cout << ">> Loading config" << std::endl;
 	if (!g_config.load()) {
 		startupErrorMessage("Unable to load config.lua!");
 		return;
 	}
 
-#ifdef _WIN32
+#	ifdef _WIN32
 	const std::string& defaultPriority = g_config.getString(ConfigManager::DEFAULT_PRIORITY);
 	if (strcasecmp(defaultPriority.c_str(), "high") == 0) {
 		SetPriorityClass(GetCurrentProcess(), HIGH_PRIORITY_CLASS);
-	} else if (strcasecmp(defaultPriority.c_str(), "above-normal") == 0) {
+	}
+	else if (strcasecmp(defaultPriority.c_str(), "above-normal") == 0) {
 		SetPriorityClass(GetCurrentProcess(), ABOVE_NORMAL_PRIORITY_CLASS);
 	}
-#endif
+#	endif
 
-	//set RSA key
+	// Load RSA key from '/key.pem' to 'g_RSA'
 	try {
 		g_RSA.loadPEM("key.pem");
-	} catch(const std::exception& e) {
+	} catch (const std::exception & e) {
 		startupErrorMessage(e.what());
 		return;
 	}
 
+	// TODO DOC
 	std::cout << ">> Establishing database connection..." << std::flush;
-
 	if (!Database::getInstance().connect()) {
 		startupErrorMessage("Failed to connect to database.");
 		return;
 	}
-
 	std::cout << " MySQL " << Database::getClientVersion() << std::endl;
-
-	// run database manager
 	std::cout << ">> Running database manager" << std::endl;
 
+	// TODO DOC
 	if (!DatabaseManager::isDatabaseSetup()) {
 		startupErrorMessage("The database you have specified in config.lua is empty, please import the schema.sql to your database.");
 		return;
 	}
 	g_databaseTasks.start();
-
 	DatabaseManager::updateDatabase();
 
 	if (g_config.getBoolean(ConfigManager::OPTIMIZE_DATABASE) && !DatabaseManager::optimizeTables()) {
 		std::cout << "> No tables were optimized." << std::endl;
 	}
 
-	//load vocations
+	// Load vocations from '/XML/vocations.xml'
 	std::cout << ">> Loading vocations" << std::endl;
 	if (!g_vocations.loadFromXml()) {
 		startupErrorMessage("Unable to load vocations!");
 		return;
 	}
 
-	// load item data
+	// TODO DOC
```

Comment:
Stripping comment in favor of a placeholder comment?
Shouldn't the todo list be resolved before you commit a PR? Especially if each PR represent a file/section already.

Reply:
Hey! Thanks for giving your feedback.

The stripped comment states the obvious and does not provide any additional insight.

I marked those as "TODO" because I wanted contributors (including myself) to know what are the parts that can receive more attention (including my own).
Also, would you leave a comment on something that you don't fully understand? I'd rather mark it.

I mentioned in the forums that I'm focusing on the more fundamental parts of the code base right now.
I intend update those comments as I read their respective files, I simply can't do it in a heartbeat...